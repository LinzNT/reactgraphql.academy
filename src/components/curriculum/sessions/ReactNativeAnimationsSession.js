import React from 'react'
import Ul, { Li } from '../../layout/Ul'
import Session from './Session'

const ReactNativeAnimationsSession = ({ title }) => (
  <Session title={title}>
    <Ul>
      <Li>Basic Animations</Li>
      <Li>Animated API</Li>
      <Li>Linear animations, Interpolations, Easings, sequence animations</Li>
      <Li>Event Drivers</Li>
      <Li>Animated API limitations</Li>
    </Ul>
  </Session>
)

export default ReactNativeAnimationsSession
