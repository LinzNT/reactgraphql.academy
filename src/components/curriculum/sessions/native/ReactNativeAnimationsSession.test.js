import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import ReactNativeAnimationsSession from './ReactNativeAnimationsSession'

describe('<ReactNativeAnimationsSession />', () => {
  it('should render component', () => {
    const wrapper = shallow(<ReactNativeAnimationsSession />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
