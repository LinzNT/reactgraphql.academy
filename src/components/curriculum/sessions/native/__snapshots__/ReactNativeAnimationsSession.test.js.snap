// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactNativeAnimationsSession /> should render component 1`] = `
<Session>
  <styled.ul>
    <styled.li>
      Basic Animations
    </styled.li>
    <styled.li>
      Animated API
    </styled.li>
    <styled.li>
      Interpolations, Linear animations, and Easings
    </styled.li>
    <styled.li>
      Composing animations
    </styled.li>
  </styled.ul>
</Session>
`;
